_data/cryptocurrencies.yml

```diff
@@ -392,6 +392,13 @@ websites:
       software: Yes
       doc: https://support.uphold.com/hc/en-us/articles/360024042471
 
+    - name: Wirex
+      url: https://wirexapp.com/
+      img: wirex.png
+      tfa: Yes
+      software: Yes
+      doc: https://wirexapp.com/help/article/how-to-enable-enhanced-security-2fa-for-my-wirex-account-0021
```

Comment:
Do we want to use short URLs for doc? https://wirexapp.com/hc/articles/207160259